src/popup/components/Modals/TransferReceive.vue

```diff
@@ -2,7 +2,7 @@
   <Modal
     class="transfer-receive-modal"
     has-close-button
-    full-screen
+    from-bottom
```

Comment:
@pstachel Can you eloborate on that?
https://github.com/aeternity/superhero-wallet/commit/9c8dd0bffc0bab3f3e945aed7d67218060f00087#diff-d6de021fcbbb90f085a88939079166a4ecc2411dee451a285f308e1fa3e8af1cR5

Reply:
This was already discussed on priv. Paweł made a temporary change on his branch and forgot to bring back the correct prop. I'm fixing this here.